packages/manager/cypress/e2e/databases/database-configuration.ts

```diff
@@ -0,0 +1,59 @@
+import { ClusterSize, Engine } from '@linode/api-v4/types';
+import { randomIp, randomLabel } from 'support/util/random';
```

Comment:
Looks like we can get rid of `randomIp` here!

```suggestion
import { randomLabel } from 'support/util/random';
```